lib/assessment.js

```diff
@@ -39,7 +55,12 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // The function you write should return
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
-
+  if (obj) {
+    this.propertyName = method;
+  } else {
+      var newObj = {};
+      newObj.propertyName = method;
+  }
```

Comment:
Almost! Dot notation won't work here - you'll just end up with a property that's literally called `propertyName`. Instead, you need to use square bracket notation.